search-sentiments-map/public/scripts/map-script.js

```diff
@@ -106,14 +107,14 @@ function updateLegends() {
  * corresponding data on the map.
  */
 function loadRegionDataByMode() {
-  isSentimentMode = !document.getElementById('sentiment-popularity-check').checked;
-
   const topicHeader = document.getElementById('topic-header');
   if (isWorldLevel) {
+    isSentimentMode = !document.getElementById('sentiment-popularity-check').checked;  
     topicHeader.innerText = isSentimentMode ?
         'Worldwide sentiment scores of search results for "' + getCurrentSearchData().topic + '"' :
         'Worldwide search interest scores for "' + getCurrentSearchData().topic + '"';
   } else {
+    isSentimentMode = false;  // Sentiment mode is not available at US level.    
     topicHeader.innerText = 
         'State-level search interest scores for "' + getCurrentSearchData().topic + '"';
   }
```

Comment:
Just checking, but if they are viewing the US mode, does the toggle for sentiment/popularity mode disappear? If not, can we either disable it or make it disappear?

Reply:
Yes it will disappear in my next PR